src/components/Clothing.js

```diff
@@ -1,11 +1,69 @@
-import styled from 'styled-components/macro'
+import { useState } from 'react'
+import styled, { css } from 'styled-components/macro'
 
-function Clothing({ imageUrl }) {
-  return <StyledImage src={imageUrl} alt="" />
+function Clothing({ titel, color, pattern, material, fitting, imageUrl }) {
+  const [showDetails, setShowDetails] = useState(false)
+
+  return (
+    <StyledSection>
+      <StyledButton
+        showDetails={showDetails}
+        onClick={() => {
+          if (showDetails) {
+            setShowDetails(false)
+          } else {
+            setShowDetails(true)
+          }
+        }}
+      >
+        <StyledImage src={imageUrl} alt="" />
+        {showDetails ? (
+          <StyledWrapper>
+            <h2>{titel}</h2>
+            <p>Farbe: {color}</p>
+            <p>Muster: {pattern}</p>
+            <p>Material: {material}</p>
+            <p>Passform: {fitting}</p>
+          </StyledWrapper>
+        ) : (
+          <div></div>
+        )}
+      </StyledButton>
```

Comment:
The semantic and HTML issue I see here is that you are adding all your details, which contain BLOCK elements, in your `button` which is an INLINE element. In HTML we should never add block elements within an inline element. I think it would be best to only wrap the image and not the whole details. 
```suggestion
      <StyledButton
        showDetails={showDetails}
        onClick={() => {
          if (showDetails) {
            setShowDetails(false)
          } else {
            setShowDetails(true)
          }
        }}
      >
        <StyledImage src={imageUrl} alt="" />
      </StyledButton>
        {showDetails && (
          <StyledWrapper>
            <h2>{titel}</h2>
            <p>Farbe: {color}</p>
            <p>Muster: {pattern}</p>
            <p>Material: {material}</p>
            <p>Passform: {fitting}</p>
          </StyledWrapper>
        )}
```